src/resources/resources-view.glade

```diff
@@ -1,26 +1,174 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.20.0 -->
+<!-- Generated with glade 3.22.1 -->
```

Comment:
Oh fish. Debian stretch is still on 3.20 .. What distro/version are you on? We need to freeze on a glade version. Probably just build-install a specific version.